File: exercises/python_playwright/navbar.py
from playwright.sync_api import Page, expect

class Navbar:
    def __init__(self, page: Page):
        self.page = page
        self.navbar = page.locator(".navbar-brand")
        self.footer = page.locator("#footer-content")

    def page_loaded(self) -> None:
        expect(self.navbar).to_be_visible()
        expect(self.footer).to_be_visible()

File: exercises/javascript_playwright/navbar.js
import { expect } from "@playwright/test";

export class Navbar {
    /**
     *
     * @param {import('@playwright/test').Page} page
     */
    constructor(page) {
        this.page = page;
        this.navbar = page.locator(".navbar-brand");
        this.footer = page.locator("#footer-content");
    }

    async pageLoaded() {
        expect(this.navbar).toBeVisible();
        expect(this.footer).toBeVisible();
    }
}
